main.js

```diff
@@ -158,6 +160,18 @@ app.on('do-update-check', (_requestFromUser) => {
   mixlibInstallUpdater.checkForUpdates(workstation.getVersion());
 });
 
+app.on('user-config-update', () => {
+  if (workstation.areUpdatesEnabled()) {
```

Comment:
Other things can change here, can we include the changed attribute in the event and drive off of that instead?

Reply:
good call.